feat: add gradle multiplatform experimental syntax (#1424)
JetBrains/kotlin-web-site: pages/docs/reference/experimental.md

}
```

> If your Gradle module is multiplatform module, you have to use `useExperimentalAnnotation` method:

Reviewer: ```suggestion
If your Gradle module is a multiplatform module, use the `useExperimentalAnnotation` method:
```

```groovy

> sourceSets {

Reviewer: Please add a multilanguage sample (as in the example above) with all `<div>`s and language names. 
In Kotlin DSL, use the `"kotlin.Experimental"` instead of `'kotlin.Experimental'` (double quotation marks instead of single).
